plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -914,6 +915,20 @@ public boolean configure(final String name, final Map<String, Object> params) th
             throw new CloudRuntimeException(e.getMessage());
         }
 
+        // destroy default network, see https://libvirt.org/sources/java/javadoc/org/libvirt/Network.html
+        try {
+            Network network = conn.networkLookupByName("default");
+            s_logger.debug("Found libvirt default network, destroying it and setting autostart to false");
+            if (network.isActive() == 1) {
+                network.destroy();
+            }
+            if (network.getAutostart()) {
```

Comment:
If we destroy the network and set autostart to false, would that cause an issue?

Reply:
@rhtyd We have destroyed the default network and did not see any issue in our production.
it seems to break packaging/debian/cloudstack-agent.init, however, as /etc/init.d/cloudstack-agent is not in use any more after systemd changes, it should not be a problem.

Reply:
We still do support initd with centos6 packages.

Reply:
@rhtyd 
I checked the code again, the packaging/debian/cloudstack-agent.init will be packaged in ubuntu/debs and deployed on ubuntu servers as /etc/init.d/cloudstack-agent. 
Since we do not support ubuntu 12.04 any more , and systemd is used in ubuntu 14.04 and later, I think this change will not break other functions.

Reply:
LGTM